gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -1,5 +1,71 @@
 """Script to create release sites HT for v4 genomes."""
 import hail as hl
+from gnomad.utils.vep import filter_vep_transcript_csqs
+
+# TODO: steps to create release sites HT
+# input: VRS-annotated sites HT v3.1.4
+# move SIFT and PolyPhen to insilico_predictors in vep_ht
+# remove missing fields in vep_ht
+# drop vep from site HT and merge with vep_ht
+# remove old insilico predictors and replace with new ones (the left predictors)
+# update dbSNP to dbSNP 156
+# update freq, freq_meta, freq_index_dict by integrating updates in HGDP/TGP
+# rerun inbreeding_coefficient with the callstats from the new freq
+# replace oth with remaining in global fields
+# update global fields
```

Comment:
I think this would be better as tickets in zenhub? We could track and have a history that way.

Reply:
Yeah, I put this in my epic.